subsys/usb/usb_device.c

```diff
@@ -152,6 +147,15 @@ struct usb_setup_packet_packed {
 	uint16_t wLength;
 } __packed;
 
+static int usb_handle_standard_request(struct usb_setup_packet *setup,
+				       int32_t *len, uint8_t **data_buf);
+static int class_handler(struct usb_setup_packet *pSetup,
+			  int32_t *len, uint8_t **data);
+static int vendor_handler(struct usb_setup_packet *pSetup,
+			  int32_t *len, uint8_t **data);
+static int custom_handler(struct usb_setup_packet *pSetup,
+			  int32_t *len, uint8_t **data);
+
```

Comment:
This could actually be avoided, don't You want to do that?

Reply:
I agree, it is temporary, I did not want to do it yet. There is still some stuff to clean up, then we can reorganize it more logical.